index.mdx

```diff
@@ -38,6 +38,37 @@ export default LandingContainer;
   <Header2>Explore open source projects currently in development.</Header2>
 </Jumbotron>
 
+<Box p={{ _: "8px", sm: "12px", md: "16px", lg: "24px" }}>
+  <FlexSection justifyContent="space-around" flexWrap="wrap">
+    <WideTile 
+      style={{ height: "284px"}}
+      to="docs/axelar/intro-to-axelar.md"
+      header="Axelar Bridge"
+      color="#141A1F"
+      textAlign="left"
+      disableArrow
+    >
+      <TileText>
+        The Axelar Network is a Web3 interoperability platform.
```

Comment:
where can I find this text in the preview?

Reply:
This is available on the main landing pages tiles, viewable with this preview: https://preview.redoc.ly/opensource-ripple/axelar-preview/